### PR TITLE
Create issue templates for bugs and features

### PR DESCRIPTION
**Pull request by Zanadar**
Creates a standard template for bug reports and feature requests.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Bug
+about: Bug report
+title: ''
+labels: 'needs-triage'
```
Comment (jromero) [inline .github/ISSUE_TEMPLATE/bug.md]: Could we set the following labels: `status/triage, type/bug`

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+assignees: ''
+
+---
```
Comment (jromero) [inline .github/ISSUE_TEMPLATE/bug.md]: We should add request for more info here:

Things like:

```
### Summary
<!--- Provide a general summary of the issue in the Title above. -->


---

### Reproduction

##### Steps
<!--- What steps should be taken to reproduce the issue? -->

1.
2.
3.

##### Expected
<!--- Tell us what should happen. -->


---

### Environment

##### pack info
<!--- Run `pack report` and copy output here. -->

##### docker info
<!--- Run `docker info` and copy output here. -->

```

**Comment by Zanadar** [comment]
Don't know if this should be an RFC or not @sclevine still kind of figuring out what does and doesn't need an RFC.

File: .github/ISSUE_TEMPLATE/feature.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+name: Feature request
+about: Suggest a new feture or an improvement to existing functionality
+title: ''
+labels: type/enhancement, status/triage
+assignees: ''
+
+---
+
+**Description**
```
Comment (jromero) [inline .github/ISSUE_TEMPLATE/feature.md]: Nitpick: Might want to use headers `####` to be more visually separating given that might use bold text in their responses.

Comment (Zanadar) [reply .github/ISSUE_TEMPLATE/feature.md]: Good call to make them consistent.

**Pull request merged**